build.gradle

```diff
@@ -69,7 +69,7 @@ dependencies {
         exclude group: 'org.spockframework', module: 'spock-core'
     }
     testImplementation 'org.spockframework:spock-core:2.3-groovy-3.0'
-    testImplementation 'org.spockframework:spock-junit4:2.3-groovy-3.0'
+    testImplementation 'org.spockframework:spock-junit4:2.3-groovy-3.0' // compat layer for junit4
     testImplementation 'junit:junit:4.13.2'
```

Comment:
ok. I meant, why we use both of them. Imo we can delete the _testImplementation 'junit:junit:4.13.2'_